database/postgres/dal/game.py

```diff
@@ -34,31 +34,37 @@ def create_game(self, games_dto: GameDTO) -> Games:
             date=games_dto.date,
             api_id=games_dto.api_id
         )
-        self.db_session.add(new_game)
-        self.db_session.commit()
-        self.db_session.refresh(new_game)
+        self.session.add(new_game)
+        self.session.commit()
+        self.session.refresh(new_game)
         return new_game
 
     def get_game_by_game_id_and_api_id(self, game_id: int, game_api_id: int) -> Optional[Games]:
-        return self.db_session.query(Games).filter_by(game_id = game_id, api_id=game_api_id).first()
+        return self.session.query(Games).filter_by(game_id = game_id, api_id=game_api_id).first()
 
     def get_game_by_id(self, game_id: int) -> Optional[Games]:
-        return self.db_session.query(Games).filter_by(game_id = game_id).first()
+        return self.session.query(Games).filter_by(game_id = game_id).first()
 
     def update_game(self, game_id: int, games_dto: GameDTO) -> Optional[Games]:
         game = self.get_game_by_id(game_id)
         if not game:
             return None
         for field, value in games_dto.dict(exclude_unset=True).items():
             setattr(game, field, value)
-        self.db_session.commit()
-        self.db_session.refresh(game)
+        self.session.commit()
+        self.session.refresh(game)
         return game
 
     def delete_game(self, game_id: int) -> bool:
         game = self.get_game_by_id(game_id)
         if not game:
             return False
-        self.db_session.delete(game)
-        self.db_session.commit()
+        self.session.delete(game)
```

Comment:
empty lines again

Reply:
please add empty lines after line 64 and before line 67.

Also, let the one who wrote this know that we must maintain readability in our code